src/vcpkg.json

```diff
@@ -34,6 +34,14 @@
       "features": [
         "zlib"
       ]
+    },
+    {
+      "name": "cucumber-cpp",
+      "version>=": "0.7.0"
+    },
+    {
+      "name": "gtest",
+      "version>=": "1.11.0"
     }
```

Comment:
cucumber-cpp (optionnally) depends on gtest, so it would make more sense to have this in src/ports/cucumber-cpp/vcpkg.json in my opinion.

Reply:
thanks for the tip!

Reply:
actually both cucumber-cpp and the tests/cucumber sources (AntaresSteps.cpp) depend on GTest